### ISSUE TITLE
[BUG] Low OTG mouse sensitivity when using >125Hz polling rate

### ISSUE THREAD

**Issue by jhosep3** [PojavLauncherTeam/PojavLauncher#5647]
### Describe the bug

![Screenshot_20240622_005032_PojavLauncher (Minecraft Java Edition for Android)](https://github.com/PojavLauncherTeam/PojavLauncher/assets/26682671/a37f7be2-acd9-49a7-945f-8a3812b00343)
 no puedo quitar las teclas virtuales, juego con un teclado inalambrico y con samsung dex
 desde que se me actualizo el juego no me quiere funcionar bien la sensiblidad, se siente demasiado lento y no puedo quitar las teclas virtuales ni configurarlas 

### The log file and images/videos

![Screenshot_20240622_005032_PojavLauncher (Minecraft Java Edition for Android)](https://github.com/PojavLauncherTeam/PojavLauncher/assets/26682671/b5e28c7c-d422-4f9c-bc20-60ad54834768)

### Steps To Reproduce

```markdown
al iniciar el laucher con la actualizacion de ayer dejo de funcionar mi sensibilidad como siempre, ahora se siente demasiado lenta y no puedo editar los controles ni quitarlos
```

### Expected Behavior

me gustaria la sensibilidad, de antes iba de maravilla asi

### Platform

```markdown
- Device model: samsung zfol 3
- CPU architecture: snad 888
- Android version: 14
- PojavLauncher version: la ultima
```

### Anything else?

nada mas

**Comment by GaucovyAero** [comment]
what mouse are you using?

**Comment by Gamr13** [comment]
Same issue with any Logitech mouse I use. 1000Hz polling rate.

**Comment by GaucovyAero** [comment]
> Same issue with any Logitech mouse I use. 1000Hz polling rate.

i guess you can use logitech onboard memory manager to change it if you want

**Comment by Gamr13** [comment]
Any idea what I should change it to? 
Is there an ideal polling rate that works with Pojav?

**Comment by GaucovyAero** [comment]
> Any idea what I should change it to? Is there an ideal polling rate that works with Pojav?

125Hz but you can always just use higher DPI, just change the polling rate back when ur using a PC i guess

**Comment by Gamr13** [comment]
Fair, cheers, I'll give that a whirl

**Comment by Gamr13** [comment]
> > Any idea what I should change it to? Is there an ideal polling rate that works with Pojav?
> 
> 125Hz but you can always just use higher DPI, just change the polling rate back when ur using a PC i guess

Right, well, I can confirm 125Hz does not fix the issue, on any of my mice, Pro X, G502, G402, and Lift.

**Comment by xViaOnGithub** [comment]
Can replicate using a USB mouse with Samsung DeX (Samsung's desktop mode) on my Samsung Galaxy S22+.
Seems like PojavLauncher is permanently capturing the device pointer to manipulate the virtual mouse, instead of allowing the device pointer to be used to control the app and capturing only when Minecraft captures the mouse.

Like OP said, this behaviour was not present in previous version

Perhaps this behavior could become a toggle in settings instead of forced? Something like "Direct mouse control?"

- Device model: Samsung Galaxy S22+
- CPU architecture:  Qualcomm Snapdragon 8 Gen 1
- Android version: One UI 6.1 (Samsung's flavor of Android 14)
- PojavLauncher version: foxglove-20240727-ae48e8e-v3_openjdk

**Comment by artdeell** [comment]
This is not a proper solution as Pojav needs to capture the pointer anyways
when in game. The reason why thic change was made in the first place is to
provide consistency between the mouse in-game and in menus

чт, 22 авг. 2024 г., 07:27 xViaOnGithub ***@***.***>:

**Comment by xViaOnGithub** [comment]
> This is not a proper solution as Pojav needs to capture the pointer anyways when in game. The reason why thic change was made in the first place is to provide consistency between the mouse in-game and in menus

So what exactly is keeping the app from only capturing my mouse pointer when Minecraft does (like it did in the last version)?

This was not an issue I had in the last version, Because back then, the app didn't force the virtual pointer on devices that have a real pointer

**Comment by ofry1** [comment]
please fix unplayable at the moment
